spec/support/ansible_shared/automation_manager/configuration_script.rb

```diff
@@ -127,22 +127,13 @@
         expect(AnsibleTowerClient::Connection).to receive(:new).and_return(atc)
         expect(EmsRefresh).to receive(:queue_refresh_task).and_return([finished_task])
         expect(ExtManagementSystem).to receive(:find).with(manager.id).and_return(manager)
-
```

Comment:
No need to remove the blank line here.  It visually separates the test setup (expectations) from the method call.